packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/AddAccountModal/AddAccountModal.tsx

```diff
@@ -116,7 +123,8 @@ export const AddAccountModal = ({ device, onCancel, symbol, noRedirect }: AddAcc
              * By default it is visible publicly, but it can be remotely hidden under debug menu.
              */
             .filter(({ backendType }) => backendType !== 'coinjoin' || isCoinjoinVisible)
-            .filter(({ isDebugOnlyAccountType }) => !isDebugOnlyAccountType || isDebug);
+            .filter(({ isDebugOnlyAccountType }) => !isDebugOnlyAccountType || isDebug)
+            .filter(({ accountType }) => (isCoinjoinDisabled ? accountType !== 'coinjoin' : true));
```

Comment:
Nit: This seems fine but I think it would be better readable if there was only one `.filter` that concerns coinjoin, rather than having two. How about doing this at line 112?
`const isCoinjoinVisible = (isCoinjoinPublic || isDebug) && !isCoinjoinDisabled;`

Note: current filter filters by `backendType`, not `accountType`, but with our data this is equivalent :heavy_check_mark: